### ISSUE TITLE
[Issue] Missing import for System.Runtime.InteropServices in WebBrowserUtils.cs

### ISSUE THREAD

**Issue by jlsalmon** [Voltstro-Studios/UnityWebBrowser#366]
### What platform are you experiencing this issue on?

MacOS arm64

### What version of UWB are you using?

2.2.4

### What Unity version are you running?

2022.3.28f1

### Describe what the issue you are experiencing is.

Hello,

I just started using UWB, and I ran into a compilation issue straight away (I develop on macOS).

It seems like there's a missing import for `System.Runtime.InteropServices` in `WebBrowserUtils.cs`. I needed to add `using System.Runtime.InteropServices;` to make the project compile.

### Provide reproducible steps for this issue.

1. Install UnityWebBrowser in a project running on macOS
2. Observe the following compilation errors:

```
Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.4/Runtime/Helper/WebBrowserUtils.cs(150,20): error CS0103: The name 'RuntimeInformation' does not exist in the current context
Library/PackageCache/dev.voltstro.unitywebbrowser@2.2.4/Runtime/Helper/WebBrowserUtils.cs(150,62): error CS0103: The name 'Architecture' does not exist in the current context
```

**Comment by Voltstro** [comment]
MacOS uses System.Runtime.InteropServices for calling `RuntimeInformation.ProcessArchitecture`, at some point of time the using got removed since the auto cleanup was not done on MacOS.

Fixed in release 2.2.5. This particular case shouldn't happen again as I embedded the namespace on the line.

**Comment by jlsalmon** [comment]
Thanks for the quick response and the release @Voltstro 👍
